client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -33,6 +34,49 @@ import QRCode from './QRCode'
 import Prose from './Prose'
 import Text from './Text'
 
+const qrCodeTargetClassName = 'qr-code-target'
+
+interface PagedJSPage {
+  element: {
+    dataset: {
+      pageNumber: any
```

Comment:
From your screenshot it looked like this would be `string`.

Reply:
D'oh! Yes.